ParisBNF/abb/BNFabb147.xml

```diff
@@ -234,17 +237,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                       </item>
 
                       <item xml:id="a2">
-                      <locus from="127b"/>
+                      <locus target="#66rb">127b</locus>
```

Comment:
p 127 stands for fol. 64r, if we start from 1 (66r is only correct if i-ii are renamed as 1-2)